.env_example

```diff
@@ -9,5 +9,8 @@ LN_CLIENT_TYPE=
 # Root key for minting macaroons
 ROOT_KEY=
 
-TEST_MACAROON=
-TEST_PREIMAGE=
+TEST_MACAROON_VALID=
```

Comment:
I think it is better to just hard-code these macaroon values, they are not sensitive info.